Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN --mount=type=bind,source=.,target=.,readonly\
 
 FROM debian:bookworm-slim AS wardend
 RUN apt update && \
-    apt install ca-certificates -y && \
+    apt install ca-certificates curl -y && \
```

Comment:
I think we need `tar` ?

Reply:
We don't need install the tar. Image already contains the tar.